eng/pipelines/typespec-ci.yml

```diff
@@ -49,6 +49,11 @@ jobs:
       Folder: specification/cognitiveservices/OpenAI.Inference
       DisplayName: OpenAI.Inference
 
+  - template: templates/steps/typespec-ci.yml
+    parameters:
+      Folder: specification/servicenetworking/resource-manager/Microsoft.ServiceNetworking/cadl
```

Comment:
We need to get the "cadl" name out of the folder structure when as we fix this project.